src/slo/x_days_sli/sli_table_recreation_filter.py

```diff
@@ -0,0 +1,28 @@
+import datetime
```

Comment:
Why this class is called filter? Is it filtering or checking? :) Please rename both SLITableRecreationFilter and SLITableExistsFilter to ... Checker? Test? Predicate ?

Reply:
done